tests/wpunit/TemplateTest.php

```diff
@@ -111,4 +111,63 @@ static function ( string $_template_file, bool $require_once, array $args ) use
 		$this->assertSame( false, $require );
 		$this->assertSame( $template->get_args( 'telemetry-library' ), $arguments );
 	}
+
+	public function test_get_option_name() {
+		$template = Config::get_container()->get( Opt_In_Template::class );
+
+		$this->assertSame( 'stellarwp_telemetry_telemetry-library_show_optin', $template->get_option_name( 'telemetry-library' ) );
```

Comment:
You might be able to make a few of these strings constants just to ensure that the spelling doesn't cause a problem sometime in the future if we change something.

Reply:
This test assures us that the spelling is exactly the same as what we expect. Wouldn't we want the test to fail if the spelling is changed in the future?

Reply:
I was explicitly referring to `telemetry-library`, which is referenced quite a few times in this file and will always be `telemetry-library` or at least the same string for all of those items if it were to change.